src/WebJobs.Script/Rpc/FunctionRegistration/FunctionDispatcher.cs

```diff
@@ -56,7 +55,6 @@ internal class FunctionDispatcher : IFunctionDispatcher
             ILanguageWorkerChannelFactory languageWorkerChannelFactory,
             IOptions<LanguageWorkerOptions> languageWorkerOptions,
             IWebHostLanguageWorkerChannelManager webHostLanguageWorkerChannelManager,
-            IJobHostLanguageWorkerChannelManager jobHostLanguageWorkerChannelManager,
```

Comment:
I think it is still worth it keep JobHostLanguageWorkerChannelManager to maitain the life time of language worker channels that start as part of function dispatcher initialization instead of creating language worker process at webhost from with in the jobhost scope.